README.md

```diff
@@ -1,6 +1,10 @@
-# tedana
+tedana: TE Dependent ANAlysis
+=============================
 
-`TE`-`de`pendent `ana`lysis (_tedana_) is a Python module for denoising multi-echo functional magnetic resonance imaging (fMRI) data.
+The ``tedana`` package is part of the ME-ICA pipeline, performing TE-dependent
+analysis of multi-echo functional magnetic resonance imaging (fMRI) data.
+``TE``-``de``\pendent ``ana``\lysis (``tedana``) is a Python module for denoising
```

Comment:
Can we remove the escape characters ('\') here ? They're being rendered as text.